syntax-across-languages.html.pl

```diff
@@ -4222,7 +4224,7 @@ package body p is
   'exponentiation vs negation (is -3^2 equal to 9 or -9)' =>
   [ { MLANG => 'C C++ MUMPS Go' },
    'mathematical' => "Perl Perl6 Maple Mathematica Io Eiffel Ruby Python Haskell Matlab ClassicREXX",
-   'negation first' => "OCaml F#",
+   'negation first' => "OCaml F# JavaScript",
```

Comment:
It seems it exponention and negation can not be used in the same expression without explicit priorities (with parentheses). Firefox says:

> unparenthesized unary expression can't appear on the left-hand side of '**'

Reply:
removed it.